clouds/clouds-core/src/main/java/sunstone/api/WithAwsCfTemplate.java

```diff
@@ -33,7 +33,7 @@
      * Region that should be used for creating resource group. Expression is allowed, e.g. {@code abc-${var:default}-xyz} -
```

Comment:
`WithAwsCfTemplate`

To be honest I wouldn't expect keywords `Aws`  or `Azure` in core module.